src/pages/LandingPage.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { Box } from '@mui/material'; 
+import { useNavigate } from 'react-router-dom'; 
+import logo from '../assets/logo.png'; 
+
+const LandingPage = () => {
+  const navigate = useNavigate(); 
+
+  const handleAnimationEnd = () => {
+    navigate('/home'); 
+  };
+
+  useEffect(() => {
+    const timeout = setTimeout(handleAnimationEnd, 3000); 
```

Comment:
This looks good. One thing you might consider in the future is using the built-in React Hook useTimeout instead of useEffect() here, just for more clarity, but it looks good.